LAD-AMA-Common/telegraf_utils/telegraf_config_handler.py

```diff
@@ -49,6 +49,27 @@ def is_systemd():
     check_systemd = os.system("pidof systemd 1>/dev/null 2>&1")
     return check_systemd == 0
 
+def is_arc_installed():
```

Comment:
This and functions below are duplicated

Reply:
I wrote separate functions for both telegraf and ME files. Would you rather us write them once and import one file into another to use? I feel like that would introduce more complication?

Reply:
Can some of these common to a common library?

Reply:
Added them to a common utils lib